src/components-examples/cdk/portal/cdk-portal-overview/cdk-portal-overview-example.ts

```diff
@@ -14,17 +14,23 @@ export class CdkPortalOverviewExample implements AfterViewInit {
   selectedPortal: Portal<any>;
   componentPortal: ComponentPortal<ComponentPortalExample>;
   templatePortal: TemplatePortal<any>;
+  domPortal: DomPortal<any>;
 
-  constructor(private _viewContainerRef: ViewContainerRef) {}
+  constructor(private _viewContainerRef: ViewContainerRef) { }
 
   ngAfterViewInit() {
     this.componentPortal = new ComponentPortal(ComponentPortalExample);
-    this.templatePortal = new TemplatePortal(this.templatePortalContent, this._viewContainerRef);
+    this.templatePortal = new TemplatePortal(
+      this.templatePortalContent,
+      this._viewContainerRef
+    );
+    const domElement = document.querySelector('#domPortalContent');
```

Comment:
Instead of querying the DOM, we should use a `ViewChild` to get a reference.

Reply:
I'm actually okay with using `querySelector` specifically for `DomPortal` because, by it's very nature, gets into touching the DOM.